src/GovUk.Education.ExploreEducationStatistics.Public.Data.Processor.Tests/Functions/ProcessCompletionOfNextDataSetVersionImportFunctionTests.cs

```diff
@@ -117,6 +124,3168 @@ private static Mock<TaskOrchestrationContext> DefaultMockOrchestrationContext(Gu
+                change: locationOptionMetaChanges[0]);
+            AssertLocationOptionChanged(
+                expectedOldOptionLink: oldLocationMetas[GeographicLevel.EnglishDevolvedArea][SqidEncoder.Encode(3)],
+                expectedNewOptionLink: newLocationMetas[GeographicLevel.EnglishDevolvedArea][SqidEncoder.Encode(3)],
+                change: locationOptionMetaChanges[1]);
+            AssertLocationOptionDeleted(
+                expectedOptionLink: oldLocationMetas[GeographicLevel.Country][SqidEncoder.Encode(5)],
+                change: locationOptionMetaChanges[2]);
+            AssertLocationOptionDeleted(
+                expectedOptionLink: oldLocationMetas[GeographicLevel.EnglishDevolvedArea][SqidEncoder.Encode(1)],
+                change: locationOptionMetaChanges[3]);
+            AssertLocationOptionChanged(
+                expectedOldOptionLink: oldLocationMetas[GeographicLevel.Country][SqidEncoder.Encode(7)],
+                expectedNewOptionLink: newLocationMetas[GeographicLevel.Country][SqidEncoder.Encode(7)],
+                change: locationOptionMetaChanges[4]);
+            AssertLocationOptionDeleted(
+                expectedOptionLink: oldLocationMetas[GeographicLevel.Country][SqidEncoder.Encode(6)],
+                change: locationOptionMetaChanges[5]);
+            AssertLocationOptionChanged(
+                expectedOldOptionLink: oldLocationMetas[GeographicLevel.Country][SqidEncoder.Encode(8)],
+                expectedNewOptionLink: newLocationMetas[GeographicLevel.Country][SqidEncoder.Encode(8)],
+                change: locationOptionMetaChanges[6]);
+            AssertLocationOptionChanged(
+                expectedOldOptionLink: oldLocationMetas[GeographicLevel.EnglishDevolvedArea][SqidEncoder.Encode(4)],
+                expectedNewOptionLink: newLocationMetas[GeographicLevel.EnglishDevolvedArea][SqidEncoder.Encode(4)],
+                change: locationOptionMetaChanges[7]);
+            AssertLocationOptionAdded(
+                expectedOptionLink: newLocationMetas[GeographicLevel.EnglishDevolvedArea][SqidEncoder.Encode(10)],
+                change: locationOptionMetaChanges[8]);
+            AssertLocationOptionAdded(
+                expectedOptionLink: newLocationMetas[GeographicLevel.Country][SqidEncoder.Encode(11)],
+                change: locationOptionMetaChanges[9]);
+            AssertLocationOptionAdded(
+                expectedOptionLink: newLocationMetas[GeographicLevel.EnglishDevolvedArea][SqidEncoder.Encode(9)],
+                change: locationOptionMetaChanges[10]);
+            AssertLocationOptionAdded(
+                expectedOptionLink: newLocationMetas[GeographicLevel.Country][SqidEncoder.Encode(12)],
+                change: locationOptionMetaChanges[11]);
+        }
+
+        private static void AssertSingleLocationDeleted(
+            IReadOnlyList<LocationMetaChange> changes,
+            LocationMeta expectedLocationMeta)
```

Comment:
Could simplify parameter to `expectedLocation`.

Similar applies for other assertion methods below.